api-examples/pretrain_tlm_pytorch.py

```diff
@@ -102,7 +102,12 @@ def train():
         args.device, updated_local_rank = init_distributed(args.local_rank)
         args.local_rank = updated_local_rank
 
-    reader_type = "lang" if not args.preprocessed else "preprocessed"
+    if args.pattern == '*.tfrecord':
```

Comment:
In the TF version, I used a file_type, I think that makes more sense than the way we are doing it here now

Reply:
to get the pattern from that is easy
```
    pattern = os.path.join(directory, f'*.{file_type}')
```